### DISCUSSION TITLE
Feature Request: Add responsive classes to Collapse

### DISCUSSION THREAD

**Discussion by zaif3r** [saadeghi/daisyui#1649] [Ideas / feature requests]
Add the following responsive classes to Collapse component:

- collapse-xs
- collapse-sm
- collapse-md
- collapse-lg

**Comment by saadeghi** [comment]
`collapse` does not come with any sizing design decisions. you can change the font size class names or other spacings:  
https://play.tailwindcss.com/Owf2iurFli
